libs/text-to-speech/sim/tts.ts

```diff
@@ -0,0 +1,78 @@
+namespace pxsim.tts {
+    //%
+    export function _getLanguageCode() {
+        return window.navigator.language;
+    }
+
+    //% promise
+    export function _speakAsync(text: string, pitch?: number, rate?: number, volume?: number, language?: string, onStart?: RefAction, onBoundary?: RefAction): Promise<void> {
+        return new Promise((resolve, reject) => {
+            const utterance = new SpeechSynthesisUtterance(text);
+            utterance.voice = getVoiceForLanguage(language || _getLanguageCode());
```

Comment:
Is it good to default to `window.navigator.language`? It definitely seems good for ease of use but it feels like it could make shared games messed up / I guess I'm just not sure what behavior would be expected, feel free to ignore

Reply:
I'm thinking of this as more of an accessibility feature than a game feature, so I think system language is the right thing to do here because i believe that's how screen readers work. I did add a string argument to hardcode it to a specific iso language code (not super discoverable though)